next.config.mjs

```diff
@@ -16,9 +19,9 @@ const config = {
    *
    * @see https://github.com/vercel/next.js/issues/41980
    */
-  i18n: {
-    locales: ["en"],
-    defaultLocale: "en",
-  },
+  // i18n: {
+  //   locales: ["en"],
+  //   defaultLocale: "en",
+  // },
```

Comment:
remove comments ?

Reply:
i kept this one specifically because of the next-included comment, but generally agree